helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -302,6 +300,76 @@ processors:
      - k8s.namespace.name
      - k8s.pod.name
      - k8s.pod.uid
+  {{- else }}
+  resource/splunk:
+    attributes:
+    - key: host.name
+      from_attribute: k8s.node.name
+      action: upsert
```

Comment:
`host.name` is already set by resource detection processor. Why it has to be overridden?

Reply:
ah i see. great. i will remove